src/index.ts

```diff
@@ -10,4 +10,5 @@ export { CompletedPayload } from "./payload/CompletedPayload";
 export { DidExecutedPayload } from "./payload/DidExecutedPayload";
 export { Payload } from "./payload/Payload";
 export { ErrorPayload } from "./payload/ErrorPayload";
-export { WillExecutedPayload } from "./payload/WillExecutedPayload";
\ No newline at end of file
+export { WillExecutedPayload } from "./payload/WillExecutedPayload";
+export { DispatcherPayloadMeta } from "./DispatcherPayloadMeta";
```

Comment:
I've exports `DispatcherPayloadMeta` for typing.
Could we only export `DispatcherPayloadMeta` as a Type?

I concern that exporting actual `DispatcherPayloadMeta` and the `isTruest` is not actual trusted property.